Microsoft.Toolkit.Uwp.Services/Services/Twitter/Tweet.cs

```diff
@@ -44,23 +45,87 @@ public string Text
         }
 
         /// <summary>
-        /// Gets or sets the extended mode.
+        /// Gets or sets full 280-character tweet text.
         /// </summary>
+        [Obsolete("This member has been deprecated by Twitter. Use Text instead.")]
         [JsonProperty("extended_tweet")]
         public TwitterExtended Extended { get; set; }
 
         /// <summary>
-        /// Gets or sets user who posted the status.
+        /// Gets or sets text of the tweet (280 characters).
         /// </summary>
-        [JsonProperty("user")]
-        public TwitterUser User { get; set; }
+        [JsonProperty("full_text")]
+        private string FullText { get; set; }
+
+        /// <summary>
+        /// Gets or sets display text range (indexes of tweet text without RT and leading user mentions)
+        /// </summary>
+        [JsonProperty("display_text_range")]
+        public int[] DisplayTextRange { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether tweet is truncated
+        /// (true when tweet is longer than 140 characters)
+        /// This entity may be deprecated - it never seems to be set to true.
+        /// </summary>
+        [JsonProperty("truncated")]
+        public bool Truncated { get; set; }
```

Comment:
I'd rename the variable as `IsTruncated` to imply that it's a boolean. Same with the others.